wasm/src/main/java/com/dylibso/chicory/wasm/WasmModuleType.java

```diff
@@ -0,0 +1,6 @@
+package com.dylibso.chicory.wasm;
+
+public enum WasmModuleType {
```

Comment:
If we remove the builder, I don't think this enum will be needed

Reply:
I think the current iteration is "good enough" as a single PR.
We can iterate over this on top.